Make it better

albus522 opened the pull request:
@mattslack how did I do?

Background changes based on actively heating, cooling, or idle.

![img_1342](https://cloud.githubusercontent.com/assets/7812/11326499/5c721c0e-9139-11e5-9319-25e6412f719e.PNG)

albus522 committed:
Add a readings model

albus522 committed:
Have the control process save readings

albus522 committed:
Add font-awesome

albus522 committed:
Use the saved readings to show current temp

albus522 committed:
Add increase/decrease temp arrows

albus522 committed:
Nope

albus522 committed:
Setup capistrano

albus522 committed:
Even more condensed controls

albus522 committed:
Stop the controls at min/max temp

albus522 committed:
Save the set temp at reading

albus522 committed:
Need static assets

mattslack committed:
Those should be buttons not links…

mattslack committed:
Actually, we could just copy those two SVGs out of font awesome, and not require a gem…

mattslack committed:
Since we're using buttons now, we cna leverage the disabled state

albus522 commented:
@mattslack your updates now trigger a double update everytime you hit up or down on iOS

mattslack commented:
Huh. Cool. I was confused on the desktop that the change event wasn't firing… interesting that iOS does it right.

albus522 committed:
Need prevent default for mobile safari

Without the prevent default mobile safari registers 2 updates. I suspect the prevent default prevents the click from registering after the touch end registers.

albus522 committed:
touchend is not prevented on disabled elements
